packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -39,6 +39,14 @@ <h2>Legacy Examples</h2>
       </fieldset>
     </form>
 
+    <form id="conversation-dialer">
```

Comment:
note: this is just a convenience, you could put the same information (convoID OR conversation url) into the normal dialer input and it would work the same